crossing-training.lic

```diff
@@ -215,6 +218,14 @@ class CrossingTraining
     @cyclic_cycle_timer = Time.now
   end
 
+  def check_attunement
+    return unless @settings.crossing_training.include?('Attunement')
+    return unless DRSkill.getxp('Attunment') <= 33
+    return unless Time.now - @attunement_timer > 60
```

Comment:
having it be exterior to exp_timers allows users to restrict the frequency at which they power walk, yet maintain some training in attunement while training other skills.